examples/tts/conf/uniglow.yaml

```diff
@@ -0,0 +1,97 @@
+name: &name "UniGlow"
```

Comment:
Can you resolve conflicts?

Reply:
To which conflicts are you referring in `uniglow.yaml`?